Voycar.Api.Web.Tests.Integration/Users/Post_ResetPassword/Tests.cs

```diff
@@ -0,0 +1,168 @@
+namespace Voycar.Api.Web.Tests.Integration.Users.Post_ResetPassword;
+
+using Context;
+using Microsoft.EntityFrameworkCore;
+using Setup;
+using R = Features.Users.Endpoints.Post.ResetPassword;
+using L = Features.Users.Endpoints.Post.Login;
+
+
+public class Tests : TestBase<App>
+{
+    private readonly App _app;
+    private readonly VoycarDbContext Context;
+
+
+    // Setup request client
+    public Tests(App app)
+    {
+        this._app = app;
+        this.Context = this._app.Context;
+    }
+
+
+    [Fact]
+    public async Task Post_Request_AsMember_ReturnsOk_And_ResetPassword()
+    {
+        // Arrange
+        const string email = "member2@test.de";
+        var memberClient = await ClientFactory.CreateMemberClient(this._app, this.Context, email, "password");
+        var request = new R.Request
+        {
+            PasswordResetToken = "F01E0980-AB43-47E2-9300-DA69838E8797s", Password = "newPassword"
+        };
+
+        var userInDb = this.Context.Users.First(user => user.Email == email);
+        userInDb.PasswordResetToken = "F01E0980-AB43-47E2-9300-DA69838E8797s";
```

Comment:
Ich würde für alle Tests einfach den Token direkt aus der Request ziehen.
```suggestion
        userInDb.PasswordResetToken = request.PasswordResetToken;
```